### ISSUE TITLE
Discuss date facet usability

### ISSUE THREAD

**Issue by hackmastera** [chemheritage/chf-sufia#770]
Some confusion among those who used the date slider. Two clicked on the histogram and didn’t know how to re-expand search since the new filter showed up instead. One typed dates into boxes but didn’t know they had to click “Apply”.

**Comment by MDiMeo** [comment]
At today's check-in we discussed making an easier way to search by date outside of the date facet. @jrochkind suggested having date boxes under the search bar, to the right of the Public Domain option. The benefit is that users wouldn't have to hit the "apply" button to refine their search the way they currently do in the facet. This will need design mock-ups. We discussed having a link for "More search options" that expands to include public domain and date, or an automatic expansion without the link to reduce number of clicks. We can still leave the date facet as is.
